composer.lock

```diff
@@ -4,7 +4,7 @@
         "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies",
         "This file is @generated automatically"
     ],
-    "content-hash": "5e242b23d84049cea21e9551fed88cc1",
+    "content-hash": "7fd6292577e257a09b6b3155ad9f771b",
```

Comment:
Curious how you updated the `composer.lock`. Did you just remove it and then run `composer install`?

Reply:
Yes, I removed `vendors` and `composer.lock` and then install the dependency.